vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -80,61 +80,62 @@ The values `-97`, `-98` and `-99` are now tagged as `miss`.
 
 
 ## Recode IDs
-To ensure that the IDs remain the same across multiple subsets and describe the same person, we recode them using a template. The first step is to create the path for the template.
+To ensure that the IDs remain the same across multiple subsets and describe the same person, they are recoded using a template. First, a path for the template is created.
```

Comment:
Could a short explanation be added, why identifier variables are recoded in the first place?

Reply:
sounds good!